action.yml

```diff
@@ -1,14 +1,24 @@
 name: 'GitHub Enterprise Action'
 description: '[WORK IN PROGRESS] Run your load tests on Gatling Enterprise straight from your GitHub Action CI/CD workflows.'
 inputs:
-  milliseconds:
-    description: 'number of milliseconds to wait'
+  gatling_enterprise_url:
+    description: 'URL of your Gatling Enterprise server (needed only for self-hosted instances).'
+    required: false
+    default: 'https://cloud.gatling.io'
+  api_token:
+    description: 'A Gatling Enterprise API token with at least Start permission (we recommend storing it securely using GitHub''s encrypted secrets; see https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow).'
     required: true
-    default: '1000'
-  fail:
-    description: 'whether or not the action will fail'
+  simulation_id:
+    description: 'The ID of the simulation to run.'
     required: true
-    default: 'true'
+  extra_system_properties:
```

Comment:
Inputs are basically only strings (no YAML arrays/objects), so we need to define a format to pass extra sys props/env vars and load generators configurations. I just kept the same JSON format used in the public API, seems the simplest solution.